### PR TITLE
[exporter/coralogix] Enable component

### PR DESCRIPTION
**Pull request by ofirshmuel**
**Description:** <Describe what has changed.>

Last PR of Coralogix exporter, the client connects to Coralogix and sends traces

The changes:
- readme - remove "in deployment" from the title
- component.go - add 1 more exporter to exporters list and to imports
- global go.mod - add new exporter and "replace" of coralogix exporter
- configschema/go.mod - add new exporter and "replace" of coralogix exporter

**Link to tracking Issue:** <Issue number if applicable>

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

**Review by oded-dd — APPROVED** [review]

**Comment by ofirshmuel** [comment]
Hi @codeboten,
How can I fix the "unittest" fail: https://github.com/open-telemetry/opentelemetry-collector-contrib/runs/4939624595?check_suite_focus=true#step:8:14

**Comment by jpkrohling** [comment]
This is currently failing with:

```
Error: ../../internal/components/components.go:35:2: missing go.sum entry for module providing package github.com/open-telemetry/opentelemetry-collector-contrib/exporter/coralogixexporter (imported by github.com/open-telemetry/opentelemetry-collector-contrib/internal/components); to add:
FAIL	github.com/open-telemetry/opentelemetry-collector-contrib/cmd/configschema [setup failed]
	go get github.com/open-telemetry/opentelemetry-collector-contrib/internal/components@v0.42.0
?   	github.com/open-telemetry/opentelemetry-collector-contrib/cmd/configschema/docsgen	[no test files]
```

File: CHANGELOG.md
Comment (jpkrohling) [inline CHANGELOG.md:7]: You can unify both entries in one. Also, it's missing the "#" in front of the PR number.

File: go.mod
Comment (jpkrohling) [inline go.mod:13]: Don't forget to also add this exporter to the contrib release: https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/manifest.yaml

**Review by jpkrohling — COMMENTED** [review]
Can you also confirm everything we have in the checklist is available for this component? 

There are things that we might have added after you sent in your first PR: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#adding-new-components

File: go.mod
Comment (ofirshmuel) [reply go.mod]: Thanks @jpkrohling ,
https://github.com/open-telemetry/opentelemetry-collector-releases/pull/69

**Comment by jpkrohling** [comment]
Looks like I can't update this PR with the changelog changes. Please rebase this PR and I'll merge once the CI is green. Alternatively, change this PR to allow edits from maintainers.

**Comment by ofirshmuel** [comment]
@jpkrohling the conflict fixed, Can you merge this PR?

**Comment by oded-dd** [comment]
@jpkrohling any update?

**Review by codeboten — COMMENTED** [review]
Looks like there are still lint failures:

```
level=warning msg="[runner] Can't run linter goanalysis_metalinter: buildir: failed to load package : could not load export data: no export data for \"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/coralogixexporter\""
level=error msg="Running error: 1 error occurred:\n\t* can't run linter goanalysis_metalinter: buildir: failed to load package : could not load export data: no export data for \"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/coralogixexporter\"\n\n"
```

**Review by codeboten — COMMENTED** [review]
Please ensure the component is added to `cmd/configschema`'s go.mod file, see this PR as an example: https://github.com/open-telemetry/opentelemetry-collector-contrib/commit/5f01d39bad2620aa536acf7bf5c1d1be2d590219#diff-6f43a803578a41b86dd65b0786b35cf3bcb63d3c02c77175dd549652ebffed4c

**Review by jpkrohling — APPROVED** [review]

**Comment by jpkrohling** [comment]
Please ensure the tests are passing. Refer to the contribution guidelines to learn how to run the tests and linters locally. This will save you time, as you'll catch problems before they get into the hands of reviewers.

```
=== RUN   TestDefaultExporters
    exporters_test.go:427: 
        	Error Trace:	exporters_test.go:427
        	Error:      	"[{file %!s(components.getExporterConfigFn=0x55c8020) %!s(bool=false)} {jaeger %!s(components.getExporterConfigFn=0x55c7f00) %!s(bool=false)} {jaeger_thrift %!s(components.getExporterConfigFn=0x55c7dc0) %!s(bool=false)} {kafka %!s(components.getExporterConfigFn=0x55c7c60) %!s(bool=false)} {logging %!s(components.getExporterConfigFn=<nil>) %!s(bool=false)} {opencensus %!s(components.getExporterConfigFn=0x55c7ac0) %!s(bool=false)} {otlp %!s(components.getExporterConfigFn=0x55c7920) %!s(bool=false)} {otlphttp %!s(components.getExporterConfigFn=0x55c77e0) %!s(bool=false)} {parquet %!s(components.getExporterConfigFn=0x55c76c0) %!s(bool=false)} {prometheus %!s(components.getExporterConfigFn=0x55c75a0) %!s(bool=false)} {prometheusremotewrite %!s(components.getExporterConfigFn=<nil>) %!s(bool=false)} {sapm %!s(components.getExporterConfigFn=0x55c7460) %!s(bool=false)} {signalfx %!s(components.getExporterConfigFn=0x55c7260) %!s(bool=false)} {splunk_hec %!s(components.getExporterConfigFn=0x55c70e0) %!s(bool=false)} {zipkin %!s(components.getExporterConfigFn=0x55c6fc0) %!s(bool=false)} {awskinesis %!s(components.getExporterConfigFn=0x55c6ea0) %!s(bool=false)} {awsprometheusremotewrite %!s(components.getExporterConfigFn=0x55c6d60) %!s(bool=false)} {alibabacloud_logservice %!s(components.getExporterConfigFn=0x55c6b80) %!s(bool=false)} {awscloudwatch %!s(components.getExporterConfigFn=0x55c69e0) %!s(bool=false)} {awsemf %!s(components.getExporterConfigFn=0x55c6860) %!s(bool=false)} {awsxray %!s(components.getExporterConfigFn=0x55c66e0) %!s(bool=false)} {azuremonitor %!s(components.getExporterConfigFn=0x55c65a0) %!s(bool=false)} {carbon %!s(components.getExporterConfigFn=0x55c6460) %!s(bool=false)} {datadog %!s(components.getExporterConfigFn=0x55c6360) %!s(bool=false)} {dynatrace %!s(components.getExporterConfigFn=0x55c61c0) %!s(bool=false)} {elastic %!s(components.getExporterConfigFn=0x55c6060) %!s(bool=false)} {elasticsearch %!s(components.getExporterConfigFn=0x55c5ec0) %!s(bool=false)} {f5cloud %!s(components.getExporterConfigFn=0x55c5c60) %!s(bool=false)} {googlecloud %!s(components.getExporterConfigFn=<nil>) %!s(bool=true)} {honeycomb %!s(components.getExporterConfigFn=0x55c5ae0) %!s(bool=false)} {humio %!s(components.getExporterConfigFn=0x55c59a0) %!s(bool=false)} {influxdb %!s(components.getExporterConfigFn=0x55c5860) %!s(bool=false)} {loadbalancing %!s(components.getExporterConfigFn=0x55c57a0) %!s(bool=false)} {logzio %!s(components.getExporterConfigFn=0x55c5660) %!s(bool=false)} {loki %!s(components.getExporterConfigFn=0x55c5520) %!s(bool=false)} {newrelic %!s(components.getExporterConfigFn=0x55c53e0) %!s(bool=false)} {sentry %!s(components.getExporterConfigFn=0x55c5320) %!s(bool=false)} {skywalking %!s(components.getExporterConfigFn=0x55c5260) %!s(bool=false)} {stackdriver %!s(components.getExporterConfigFn=<nil>) %!s(bool=true)} {sumologic %!s(components.getExporterConfigFn=0x55c5120) %!s(bool=false)} {tanzuobservability %!s(components.getExporterConfigFn=0x55c4fe0) %!s(bool=false)} {tencentcloud_logservice %!s(components.getExporterConfigFn=0x55c4f20) %!s(bool=false)}]" should have 43 item(s), but has 42
```

**Comment by jpkrohling** [comment]
Your PR introduced a test failure, it should not.

```
=== RUN   TestDefaultExporters
    exporters_test.go:427: 
        	Error Trace:	exporters_test.go:427
        	Error:      	"[{file %!s(components.getExporterConfigFn=0x55c8d80) %!s(bool=false)} {jaeger %!s(components.getExporterConfigFn=0x55c8c60) %!s(bool=false)} {jaeger_thrift %!s(components.getExporterConfigFn=0x55c8b20) %!s(bool=false)} {kafka %!s(components.getExporterConfigFn=0x55c89c0) %!s(bool=false)} {logging %!s(components.getExporterConfigFn=<nil>) %!s(bool=false)} {opencensus %!s(components.getExporterConfigFn=0x55c8820) %!s(bool=false)} {otlp %!s(components.getExporterConfigFn=0x55c8680) %!s(bool=false)} {otlphttp %!s(components.getExporterConfigFn=0x55c8540) %!s(bool=false)} {parquet %!s(components.getExporterConfigFn=0x55c8420) %!s(bool=false)} {prometheus %!s(components.getExporterConfigFn=0x55c8300) %!s(bool=false)} {prometheusremotewrite %!s(components.getExporterConfigFn=<nil>) %!s(bool=false)} {sapm %!s(components.getExporterConfigFn=0x55c81c0) %!s(bool=false)} {signalfx %!s(components.getExporterConfigFn=0x55c7fc0) %!s(bool=false)} {splunk_hec %!s(components.getExporterConfigFn=0x55c7e40) %!s(bool=false)} {zipkin %!s(components.getExporterConfigFn=0x55c7d20) %!s(bool=false)} {awskinesis %!s(components.getExporterConfigFn=0x55c7c00) %!s(bool=false)} {awsprometheusremotewrite %!s(components.getExporterConfigFn=0x55c7ac0) %!s(bool=false)} {alibabacloud_logservice %!s(components.getExporterConfigFn=0x55c78e0) %!s(bool=false)} {awscloudwatch %!s(components.getExporterConfigFn=0x55c7740) %!s(bool=false)} {awsemf %!s(components.getExporterConfigFn=0x55c75c0) %!s(bool=false)} {awsxray %!s(components.getExporterConfigFn=0x55c7440) %!s(bool=false)} {azuremonitor %!s(components.getExporterConfigFn=0x55c7300) %!s(bool=false)} {carbon %!s(components.getExporterConfigFn=0x55c71c0) %!s(bool=false)} {datadog %!s(components.getExporterConfigFn=0x55c70c0) %!s(bool=false)} {dynatrace %!s(components.getExporterConfigFn=0x55c6f20) %!s(bool=false)} {elastic %!s(components.getExporterConfigFn=0x55c6dc0) %!s(bool=false)} {elasticsearch %!s(components.getExporterConfigFn=0x55c6c20) %!s(bool=false)} {f5cloud %!s(components.getExporterConfigFn=0x55c69c0) %!s(bool=false)} {googlecloud %!s(components.getExporterConfigFn=<nil>) %!s(bool=true)} {honeycomb %!s(components.getExporterConfigFn=0x55c6840) %!s(bool=false)} {humio %!s(components.getExporterConfigFn=0x55c6700) %!s(bool=false)} {influxdb %!s(components.getExporterConfigFn=0x55c65c0) %!s(bool=false)} {loadbalancing %!s(components.getExporterConfigFn=0x55c6500) %!s(bool=false)} {logzio %!s(components.getExporterConfigFn=0x55c63c0) %!s(bool=false)} {loki %!s(components.getExporterConfigFn=0x55c6280) %!s(bool=false)} {newrelic %!s(components.getExporterConfigFn=0x55c6140) %!s(bool=false)} {sentry %!s(components.getExporterConfigFn=0x55c6080) %!s(bool=false)} {skywalking %!s(components.getExporterConfigFn=0x55c5fc0) %!s(bool=false)} {stackdriver %!s(components.getExporterConfigFn=<nil>) %!s(bool=true)} {sumologic %!s(components.getExporterConfigFn=0x55c5e80) %!s(bool=false)} {tanzuobservability %!s(components.getExporterConfigFn=0x55c5d40) %!s(bool=false)} {tencentcloud_logservice %!s(components.getExporterConfigFn=0x55c5c80) %!s(bool=false)}]" should have 43 item(s), but has 42
        	Test:       	TestDefaultExporters
        	Messages:   	All user configurable components must be added to the lifecycle test
```

**Comment by ofirshmuel** [comment]
@jpkrohling Thanks, I fixed this issue. there is only 1 failed test but this failure is not "coralogixexporter"
https://github.com/open-telemetry/opentelemetry-collector-contrib/runs/5094074579?check_suite_focus=true#step:8:18312

**Review by jpkrohling — APPROVED** [review]

**Pull request merged**
